### PR TITLE
Add icons for EV Plugs

### PR DESCRIPTION
**Pull request by huayunh**
- Fixes PXBLUE-????
- Cleaned up the scripts to make them easier to run from root

#### Screenshots:

image
image

#### To Test:
- `yarn start`

### CODE REVIEW

**Review by jeffvg — APPROVED** [review]
nice icons

**Comment by joebochill** [comment]
5 new icons... 70 files changed 👀

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v1.7.0 (October 1, 2021)
+
+### Added
+
+-   New icons: EV Plug Type 1, EV Plug Type 2, EV Plug CCS Combo, EV Plug CHAdeMO, RFID
```
Comment (joebochill) [inline CHANGELOG.md:7]: Is the capitalization on chademo here the way we want it?

**Review by joebochill — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (huayunh) [reply CHANGELOG.md:7]: Correct.

**Pull request merged**